reporters/fail.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+var PluginError = require('gulp-util').PluginError;
+var through = require('through2');
+
+module.exports = function (opts) {
+	opts = opts || {};
+
+	// @type false|[]paths - paths to files that failed JSCS
```

Comment:
Leftover copypasta from gulp-jshint's code. Removed the type notation, but kept the comments' explanatory part.